frontend/src/App.js

```diff
@@ -149,55 +173,66 @@ function App() {
   };
 
   const handleSuccess = (data) => {
-    toast.success("SUCCESS: " + data);
+    toast.success("SUCCESS: " + data.description);
   };
 
   const handleError = (data) => {
     toast.error(data.description);
   };
 
+  const handleCreatedExperiment = (data) => {
+    toast.success("Successfully created experiment : " + data.session_id);
+  };
```

Comment:
Note that the `CREATED_EXPERIMENT` message is a synchronization message send to all connected experimenters and not a direct response to the `CREATE_EXPERIMENT` request. The answer to  `CREATED_EXPERIMENT` is a `SUCCESS` message with type: `JOIN_EXPERIMENT`. The success type `JOIN_EXPERIMENT` is also used for the  `JOIN_EXPERIMENT` request, but the description differs.
The message "Successfully created experiment" may imply that the experimenter triggered the event, which might not be the case.

Backend-API Wiki `CREATE_EXPERIMENT`: [link](https://github.com/TUMFARSynchorny/experimental-hub/wiki/Backend-API#create_experiment)
Backend-API Wiki ` JOIN_EXPERIMENT`: [link](https://github.com/TUMFARSynchorny/experimental-hub/wiki/Backend-API#join_experiment)